queries.sql

```diff
@@ -85,4 +85,18 @@ SELECT MIN(weight_kg) AS minimum_weight,MAX(weight_kg) as max_weight,species fro
 /*What is the average number of escape attempts per animal type of those born between 1990 and 2000?*/
 SELECT AVG(escape_attempts)::NUMERIC(10,2) from animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000 GROUP BY species;
 
+/*What animals belong to Melody Pond?*/
+SELECT name FROM animals INNER JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+/*List of all animals that are pokemon (their type is Pokemon).*/
+SELECT animals.name FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+/*List all owners and their animals, remember to include those that don't own any animal.*/
+SELECT full_name, name FROM owners LEFT JOIN animals ON animals.owner_id = owners.id;
+/*How many animals are there per species?*/
+SELECT COUNT(*), species.name FROM animals INNER JOIN species ON animals.species_id = species.id GROUP BY species.name;
```

Comment:
The screenshot you added for this query is incorrect.

- How many animals are there per species?

Please correct that 🛠